### PR TITLE
Camera manipulation adjustments for openrave viewer

### PR DESCRIPTION
**Pull request by Hsifnus**

### CODE REVIEW

File: plugins/qtosgrave/osgviewerwidget.cpp [hunk 1]
```diff
@@ -279,12 +279,28 @@ class OpenRAVETrackball : public osgGA::TrackballManipulator
     // make zooming faster
     bool performMovementRightMouseButton( const double eventTimeDelta, const double dx, const double dy )
     {
-        if(_posgviewerwidget->IsInOrthoMode()) {
-            _posgviewerwidget->Zoom( dy < 0 ? 0.9 : 1.1 );
-            return true;
+        bool result = osgGA::TrackballManipulator::performMovementRightMouseButton(eventTimeDelta, dx, dy*2);
+        // adjust camera perspective if zoom changed while in ortho view
+        if (_posgviewerwidget->IsInOrthoMode()) {
```
Comment (Hsifnus) [inline plugins/qtosgrave/osgviewerwidget.cpp:284]: Am debating using a variant of what is already in production, but I'll have to recompile the code first in order to be able to test.

**Comment by pablocael** [comment]
@Hsifnus 
@braineo I tested and its working, sensitivity got better and ortho mouse zoom got fixed.
I think its ready for your review.

**Review by braineo — COMMENTED** [review]
Mouse rotation is better. Orthogonal view zoom using mouse is still reversed from perspective view

**Comment by pablocael** [comment]
oh! I tested the right mouse button. You are right, my bad. Its the mouse wheel that has issue.

**Comment by braineo** [comment]
Thanks!

**Pull request merged**